src/Stratis.Bitcoin.Features.SmartContracts/Rules/SmartContractCoinViewRuleLogic.cs

```diff
@@ -252,6 +255,22 @@ public void ExecuteContractTransaction(RuleContext context, Transaction transact
             this.CheckBlockGasLimit(result.GasConsumed);
         }
 
+        /// <summary>
+        /// Get a string representation of a smart contract execution's return value.
+        /// </summary>
+        public static string GetSerializedReturnValue(object returnValue)
```

Comment:
This definitely doesn't belong in the CoinViewRule. We should continue to return the result object, and leave it up to the presentation layer (in this case, the API controller) to decide how to display structs.

Also, we will probably want to reuse this for deserializing local calls to methods where the return type is a struct.